### ISSUE TITLE
Option to include/exclude just the input or output for each cell?

### ISSUE THREAD

**Issue by rickhg12hs** [facundobatista/jupynotex#3]
The subject of the LaTeX doc may be uninterested in the code and only the output is relevant.  Is it possible to restrict what is "imported" to just the output of a cell?

**Comment by facundobatista** [comment]
How do you imagine the cell in the doc when only the output is included? with the horizontal separation bar? IOW, how do you differentiate that from a cell with input but no output? Is it worth differentiating it?

**Comment by rickhg12hs** [comment]
Yes, when both input and output are included, some type of separation would be a great option.  With only the output present, just a box would be fine, but perhaps other LaTeX framing/boxing could be used too.

**Comment by facundobatista** [comment]
I wonder if we need this *per cell* or as a generic configuration.

If it's per cell, we'd need to create "postfixes" for cell numbers/ranges.

Like, including cells `[1,2x,3-5,6-7x]` would mean that cells 1, 3, 4 and 5 are included as today, and 2, 6 and 7 with some modification (that `x` could be `i` (just input) or `o` (just output)).

**Comment by rickhg12hs** [comment]
I like the _per cell_ idea.

It's okay to refer to the same notebook multiple times in a doc, yes?
E.g.,
```latex
...
\jupynotex[1o,3,6-8o]{sample.ipynb}

Blah, blah, blah

\jupynotex[9-11o]{sample.ipynb}
...
```
